src/widgets/PrivacyWidget.php

```diff
@@ -85,6 +85,12 @@ class PrivacyWidget extends Widget
                         justify-content: space-between;
                     }';
 
+    /**
+     * @var string CSS class to be applied to the container
+     * Custom CSS class can be used to style the container
+     */
+    public $cssClass = '';
```

Comment:
If an attribute is empty you can use `public $cssClass` which is equals to `null`. Please also add a @since phpdoc property.

Reply:
Done